invenio_orcid/config.py

```diff
@@ -26,6 +26,12 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_AUTHORS_ELASTICSEARCH_INDEX_NAME = 'records-authors'
```

Comment:
Can you use search classes?

- https://github.com/inveniosoftware/invenio-search/blob/master/invenio_search/api.py#L49
- https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/config.py#L44

```
ORCID_AUTHORS_SEARCH_CLASS = ...
ORCID_RECORDS_SEARCH_CLASS = ...
```